pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -114,9 +120,11 @@ subsets:
       - transformDiaSrcCat
       - diaPipe
       - fakesMatch
+      - sampleSpatialMetrics
       - analyzeAssocDiaSrcCore
       - analyzeTrailedDiaSrcCore
       - diffimTaskCore
+      - diffimTaskPlots
```

Comment:
Good catch. I was doing this on my end for one test related to new source injection pipeline.